app/controllers/IsThisFileConfidentialController.scala

```diff
@@ -74,7 +75,15 @@ class IsThisFileConfidentialController @Inject() (
         form
           .bindFromRequest()
           .fold(
-            formWithErrors => Future.successful(BadRequest(view(formWithErrors, mode, draftId))),
+            formWithErrors => {
+              val fileName = UploadSupportingDocumentPage.get() match {
+                case Some(file) =>
+                  file.fileName.get
+                case None       =>
+                  "fileName"
```

Comment:
Maybe a different file name would be better? Something like "uploaded document" (this may look odd since its the same name as the header), or just an empty string.

Reply:
Yeah you're right, I'll find a better way to sort this